kitchen/modules/adding-machine/Cargo.toml

```diff
@@ -10,40 +10,15 @@ std = [
     'parity-scale-codec/std',
     'support/std',
     'system/std',
-    'runtime-primitives/std',
+    'sr-primitives/std',
 ]
 
-[dependencies.parity-scale-codec]
-default-features = false
-features = ['derive']
-version = '1.1.0'
+[dependencies]
+parity-scale-codec = { package = "parity-scale-codec", version = "1.1.0", features = ["derive"], default-features = false }
+support = { package = "srml-support", rev = "6ae3b6c4ddc03d4cdb10bd1d417b95d20f4c1b6e", git = "https://github.com/paritytech/substrate.git", default_features = false }
+system = { package = "srml-system", rev = "6ae3b6c4ddc03d4cdb10bd1d417b95d20f4c1b6e", git = "https://github.com/paritytech/substrate.git", default_features = false }
 
-[dependencies.support]
-default_features = false
-git = 'https://github.com/paritytech/substrate.git'
-package = 'srml-support'
-rev = '6ae3b6c4ddc03d4cdb10bd1d417b95d20f4c1b6e'
-
-[dependencies.system]
-default_features = false
-git = 'https://github.com/paritytech/substrate.git'
-package = 'srml-system'
-rev = '6ae3b6c4ddc03d4cdb10bd1d417b95d20f4c1b6e'
-
-[dependencies.runtime-primitives]
-default_features = false
-git = 'https://github.com/paritytech/substrate.git'
-package = 'sr-primitives'
-rev = '6ae3b6c4ddc03d4cdb10bd1d417b95d20f4c1b6e'
-
-[dev-dependencies.primitives]
-default_features = false
-git = 'https://github.com/paritytech/substrate.git'
-package = 'substrate-primitives'
-rev = '6ae3b6c4ddc03d4cdb10bd1d417b95d20f4c1b6e'
-
-[dev-dependencies.runtime-io]
-default_features = false
-git = 'https://github.com/paritytech/substrate.git'
-package = 'sr-io'
-rev = '6ae3b6c4ddc03d4cdb10bd1d417b95d20f4c1b6e'
+[dev-dependencies]
+sr-primitives = { package = "sr-primitives", rev = "6ae3b6c4ddc03d4cdb10bd1d417b95d20f4c1b6e", git = "https://github.com/paritytech/substrate.git", default_features = false }
```

Comment:
> used to `runtime-primitives`, and below imports used to be `primitives` and `runtime-io`

@JoshOrndorff IDK how you feel about this decision to remove the type aliasing, but I personally think that type aliasing imports like we do in all the example code is confusing for someone looking through the code. Every type alias adds some mental overhead to trace your way back through the code.

Reply:
Yeah, I agree with that sentiment, and support this change.

I also like the idea of aliasing the packages where we're demonstrating that you can easily swap one for another. In particular, I'm thinking of how I swap which runtime is included in the node like here https://github.com/substrate-developer-hub/recipes/blob/4da4f33f8518b1606904dd03235fe4c44b576263/kitchen/node/Cargo.toml#L37-L42

Also FYI, when you use the package's own name for the dependency name as you're advocating you can omit the `package` key

```diff
- sr-primitives = { package = "sr-primitives", rev = "6ae3b6c4ddc03d4cdb10bd1d417b95d20f4c1b6e", git = "https://github.com/paritytech/substrate.git", default_features = false }
+ sr-primitives = { ev = "6ae3b6c4ddc03d4cdb10bd1d417b95d20f4c1b6e", git = "https://github.com/paritytech/substrate.git", default_features = false }
```

Reply:
Yes, I think the example you provided above better justifies the usage of aliases. I'll remove the package keys now.